scanpy/preprocessing/_scrublet/__init__.py

```diff
@@ -292,41 +289,31 @@ def _run_scrublet(ad_obs, ad_sim=None):
 
     logg.info("    Scrublet finished", time=start)
 
-    if copy:
-        return adata
-    else:
-        return None
+    return adata if copy else None
```

Comment:
```suggestion
    if copy:
        return adata
    return None
```
Not a huge fan of ternary in return statement

Reply:
why? It’s much more concise and very readable, especially if all three expressions are a single word

Reply:
I guess the readable aspect is up for debate haha, but I approved so you absolutely do not have to do this.